core/ui/src/main/java/org/phoebus/ui/application/ContextMenuService.java

```diff
@@ -57,6 +57,7 @@ public List<ContextMenuEntry> listContextMenuEntries() {
      */
     public List<ContextMenuEntry> listSupportedContextMenuEntries() {
         // List of types of the current selection
+        List list = SelectionService.getInstance().getSelection().getSelections();
```

Comment:
This 'list' looks like it's not used.
Maybe left over from debugging the following line, checking what's in the list before mapping to classes?

Reply:
Not "maybe". I'll fix it.